ansible/inventories/local_development/group_vars/all

```diff
@@ -7,16 +7,20 @@ dataserver_1_internal_ip: 127.0.0.1
 deployment_environment_id: local_development
 ssl_certificate_name: nginx-selfsigned.crt
 ssl_key_name: nginx-selfsigned.key
+redirect_ssl_certificate_name: redirect-nginx-selfsigned.crt
+redirect_ssl_certificate_key_name: redirect-nginx-selfsigned.key
 web_app_github_repo_branch: test
 search_app_github_repo_branch: test
 server_domain_name: etsin-finder.local
+redirect_server_domain_name: redirect.etsin-finder.local
```

Comment:
Remember to use existing variables when possible. Such as here: ``redirect_server_domain_name: "redirect.{{ server_domain_name}}"``

Reply:
Thanks, a good practice to follow.